### PR TITLE
Use ZF with MNC

### PR DESCRIPTION
**Pull request by RonaldEnsing**
@hgaiser: This Pull Request enables the option to select ZF with MNC.

### CODE REVIEW

File: models/ZF/mnc_3stage/test.prototxt [hunk 1]
```diff
@@ -0,0 +1,554 @@
+name: "VGG16"
```
Comment (hgaiser) [inline models/ZF/mnc_3stage/test.prototxt]: Rename this to ZF please.

File: models/ZF/mnc_5stage/test.prototxt [hunk 2]
```diff
@@ -0,0 +1,857 @@
+name: "VGG16"
```
Comment (hgaiser) [inline models/ZF/mnc_5stage/test.prototxt]: Same here.

File: models/ZF/mnc_5stage/train.prototxt [hunk 3]
```diff
@@ -0,0 +1,1243 @@
+name: "VGG16"
```
Comment (hgaiser) [inline models/ZF/mnc_5stage/train.prototxt]: And here.

**Comment by hgaiser** [comment]
Except for the references to VGG16, it looks fine.

**Comment by RonaldEnsing** [comment]
Done.

**Pull request merged**